lib/git.coffee

```diff
@@ -93,7 +93,12 @@ parseDefault = (data) -> q.fcall ->
 callGit = (cmd, parser, nodatalog) ->
   logcb "> git #{cmd}"
 
-  return git(cmd, {cwd: cwd})
+  options = {cwd: cwd}
+  customPath = atom.config.get("git-control.gitCustomPath")
+  if customPath
+    options.gitExec = customPath
```

Comment:
This doesn't work (at least on windows) for paths with a space in it. This could be solved by adding a `"` before and after the custom path (so `options.gitExec = '"' + customPath + '"'`).